### ISSUE TITLE
Undead Shadow Priest not getting Devouring Plague.

### ISSUE THREAD

**Issue by frisken95** [RetroWoW/RetroWoW#302]
**Description**: I basically didnt get devouring plague although I chose undead for that reason.

**Current behavior**: Tell us what happens
Read the description.
**Expected behavior**: Tell us what should happen instead
I expected it to be there.
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Source:**
Joddi the Shadow Priest

**Comment by muted45** [comment]
Yeah I think that fear ward (dwarf priest racial) is bugged as well. I don't think any priest racials are present.

**Comment by zyat** [comment]
Go to priest trainer in Undercity or Ironforge
